src/main/java/uk/gov/beis/els/api/openapi/OpenApiPropertyCustomiser.java

```diff
@@ -169,10 +172,44 @@ private void processEnumValues(List<Annotation> annotations, Schema schema) {
                     .collect(Collectors.toList());
 
             schema.setEnum(allowedValues);
+            schema.setExample(allowedValues.get(0));
 
           } catch (Exception e) {
             throw new RuntimeException("Error processing ApiValuesFromEnum annotations", e);
           }
         });
   }
+
+  private void processExamples(List<Annotation> annotations, Schema schema) {
+    getAnnotation(annotations, io.swagger.v3.oas.annotations.media.Schema.class)
+        .ifPresent(schema1 -> {
+          String type = schema1.type();
+          // process integer and number types
+          if (type != null) {
+            if (type.equals("integer")) {
+              if (schema.getDescription().contains("pixels")) {
```

Comment:
@jamesbarnett91 I did think about creating a specific annotation for pixel fields and QR code fields (processed below) and using that to determine the example property, but that might be overkill. For now it's based on the description. Thoughts? Could also base it on field name? Either could be inconsistent.